pennylane_lightning/core/src/simulators/lightning_gpu/CMakeLists.txt

```diff
@@ -0,0 +1,110 @@
+##########################
+## Set Project version
+##########################
+cmake_minimum_required(VERSION 3.21)
+set(LOGO [=[
+░█░░░▀█▀░█▀▀░█░█░▀█▀░█▀█░▀█▀░█▀█░█▀▀░░░░█▀▀░█▀█░█░█
+░█░░░░█░░█░█░█▀█░░█░░█░█░░█░░█░█░█░█░░░░█░█░█▀▀░█░█
+░▀▀▀░▀▀▀░▀▀▀░▀░▀░░▀░░▀░▀░▀▀▀░▀░▀░▀▀▀░▀░░▀▀▀░▀░░░▀▀▀
+]=])
+message(${LOGO})
+
+project(${PL_BACKEND}
+    DESCRIPTION "Lightning-GPU bindings for PennyLane. Backed by NVIDIA cuQuantum SDK."
+    LANGUAGES CXX C CUDA
+)
+
+# Include macro and functions supporting Nvidia and cuQuantum libraries.
+include("${pennylane_lightning_SOURCE_DIR}/cmake/support_pllgpu.cmake")
+findCUDA(lightning_external_libs)
+findCustatevec(lightning_external_libs)
+
+if(ENABLE_MPI)
+    findMPI(lightning_external_libs)
+endif()
+
+set(LGPU_FILES  StateVectorCudaManaged.cpp initSV.cu
+                CACHE INTERNAL "" FORCE)
+
+
+add_library(${PL_BACKEND} STATIC ${LGPU_FILES})
+
+target_compile_options(lightning_compile_options INTERFACE "-D_ENABLE_PLGPU=1")
+
+if(ENABLE_MPI)
+    target_compile_options(lightning_compile_options INTERFACE "-D_ENABLE_MPI=1")
+endif()
+
+##########################
+## Enforce C++ Standard ##
+##########################
+set(CMAKE_CXX_EXTENSIONS OFF)
+set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+
+# Ensure the libraries can see additional libs at same level;
+# Required for external deps when loading in Python
+set(CMAKE_BUILD_RPATH_USE_ORIGIN ON)
+set(CMAKE_SKIP_BUILD_RPATH FALSE)
+set(CMAKE_BUILD_RPATH "$ORIGIN/../cuquantum/lib:$ORIGIN/../cuquantum/lib64:$ORIGIN/")
+set(CMAKE_INSTALL_RPATH "$ORIGIN/../cuquantum/lib:$ORIGIN/../cuquantum/lib64:$ORIGIN/")
+
+#########################
+## Set Default Options ##
+#########################
+option(PLLGPU_ENABLE_SANITIZER "Enable address sanitizer" OFF)
```

Comment:
If there is a global Lightning address sanitiser option, we may also want that off here, since it will not be compatible with the CUDA memory

Reply:
Let's remove it.